recipe/meta.yaml

```
@@ -9,10 +9,11 @@ source:
   sha256: 7bcaf6e7bb3faccaf306846bf30838d8e94b3ebd58e971da8c3fe426f1f9288c
 
 build:
-  number: 1
+  number: 2
   script:
     - export LDFLAGS="$LDFLAGS -fopenmp"  # [osx]
-    - {{ PYTHON }} -m pip install . --no-deps -vv
+    - cython --force --cplus src/cyaccess.pyx  # [py>310]
```

```suggestion
    - cat $(which cython)  # [py>310]
    - cython --force --cplus src/cyaccess.pyx  # [py>310]
```

Looks like `$BUILD_PREFIX/bin/cython` is being interpreted as a shell script. Maybe the shebang is faulty? Let's find out!